projects/flyout/src/modules/flyout/flyout-instance.ts

```diff
@@ -118,11 +115,6 @@ export class SkyFlyoutInstance<T> implements OnDestroy {
     });
   }
 
-  public ngOnDestroy(): void {
-    this._iteratorPreviousButtonClick.complete();
```

Comment:
Should we be completing these in the `close` method?

Reply:
I suppose that wouldn't hurt.